src/org/javarosa/core/io/Std.java

```diff
@@ -0,0 +1,32 @@
+package org.javarosa.core.io;
+
+import java.io.PrintStream;
+
+/**
+ * Used to break dependency directly on System.out and System.err.
+ */
+public class Std {
+
+    public static PrintStream out = System.out;
```

Comment:
Ok, I'm fine with this being temporary and I agree with the argument for `Std.out` instead of `Std.out()`. @dcbriccetti can you please take a quick look -- don't want to make any changes that will surprise you since I know you're looking across the codebase.

I'll file an issue to rework logging.